config/manager/manager.yaml

```diff
@@ -30,7 +30,7 @@ spec:
                 operator: In
                 values:
                 - mongodb-kubernetes-operator
-            topologyKey: "kubernetes.io/hostname"
+            topologyKey: kubernetes.io/hostname
```

Comment:
I assume this change comes from some linting check?

Reply:
Yes when we ran the update_release.py script